src/createAnimatedComponent.tsx

```diff
@@ -293,14 +293,17 @@ export default function createAnimatedComponent(
 
     _attachNativeEvents() {
       const node = this._getEventViewRef();
-      const viewTag = findNodeHandle(options?.setNativeProps ? this : node);
+      let viewTag = null; // We set it only if needed
```

Comment:
Can we also do the same in `_reattachNativeEvents` which happens on each render?